packages/sitecore-jss-angular/src/components/placeholder.component.spec.ts

```diff
@@ -602,7 +628,21 @@ describe('<sc-placeholder /> with lazy loaded modules', () => {
             ]
           ),
         ],
-        providers: [MockUrlTreeGuard, MockUnexpectedErrorGuard],
+        providers: [
+          MockUrlTreeGuard,
+          MockUnexpectedErrorGuard,
+          JssStateService,
+          {
```

Comment:
Can you double-check if this provider is needed? I believe that it's unused, we don't have such injection token